guide-us-english.tex

```diff
@@ -234,11 +234,11 @@ \subsection{Executive MetaScrum Team (EMS)}
 
 \includegraphics[width=1.0\linewidth]{ExecMetaScrum.png}
 
-The Executive MetaScrum Team holds a stakeholder alignment meeting, the \textbf{MetaScrum Event}, on a regular cadence, at least once per Sprint.
+At least once per Sprint, the Executive MetaScrum Team holds a stakeholder alignment meetng, the \textbf{MetaScrum Event}. 
 
 \begin{itemize}
-	\item Members of the Executive MetaScrum Team (or proxy) attend the MetaScrum Event wherein the Chief Product Owner presents the Product Backlog to business owners who control funding, personnel, and customer commitments. They address any changes needed to strategy, funding, resource allocation, and deployments, and collaborate with the Chief Product Owner to build an agreement on a Product Backlog that they will support until the next MetaScrum Event.
-	\item This event is the forum for Leadership, Stakeholders, and Business Owners to express their preferences or sometimes urgent demands that may cause the Chief Product Owner's backlog to be restructured.
+	\item At the MetaScrum event, decisions affecting the entire organization get made. Changing direction of the organization, having to fire 20% of all employees, killing products and restructuring the organizational backlog can all happen at this event.
+	\item The Chief Product Owner presents the Product Backlog to the MetaScrum Team, business owners who control funding, personnel, and customer commitments. They address any changes needed to strategy, funding, resource allocation, and deployments, and collaborate with the Chief Product Owner to build an agreement on a Product Backlog that they will support until the next MetaScrum Event.
 \end{itemize}
```

Comment:
Oof, this needs a more serious look at. There's confusion in EMS, MS, EMS Team, etc. I'd rather put this in a separate pull request that addresses this confusion.